forest/sycamore/tests/test_functions.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import os
+
+from forest.sycamore.survey_config import get_all_interventions_dict
+from forest.sycamore.functions import aggregate_surveys
+from forest.sycamore.survey_config import gen_survey_schedule
+
+# Needed so that relative paths in the below file will work
+os.chdir(os.path.join("forest", "sycamore", "tests"))
+
+
+def test_get_interventions():
+    empty_dict = get_all_interventions_dict("empty_intervention_data.json")
+
+    assert empty_dict == {}
+
+    interventions_dict = get_all_interventions_dict(
```

Comment:
Each test should test one thing. I think this one can be split in two here, something like: `test_get_interventions_with_empty_input()` and `test_get_interventions_with_normal_input()`

Reply:
Thanks! This is updated :)